portfolio/src/main/webapp/script.js

```diff
@@ -72,12 +72,16 @@ async function getComments() {
     const text = await response.json();
     for (const currentText of text) {
```

Comment:
This API should send more information in the response, JSON of the format below or similar would be preferable. Don't rely on the caller knowing the convention or protocol, make it as easy as possible to consume your API response.

```json
{
  "text": "some comment",
  "imageUrl": "http://image.com/cats.png"
}
```

Reply:
Nice